packages/amplication-client/src/Workspaces/WorkspaceSelector.tsx

```diff
@@ -61,7 +61,12 @@ function WorkspaceSelector() {
               <span className={`${CLASS_NAME}__current__name`}>
                 {currentWorkspace.name}
               </span>
-              <span className={`${CLASS_NAME}__current__plan`}>
+              <span
+                className={classNames(
+                  `${CLASS_NAME}__current__plan`,
+                  currentWorkspace.subscription?.subscriptionPlan?.toLocaleLowerCase()
+                )}
+              >
```

Comment:
it looks good, but I suggest adding a fallback

Reply:
what kind of? there is a default color for this text. In case subscription plan doesn't exist, additional class just won't show up.